allennlp/semparse/contexts/table_question_context.py

```diff
@@ -115,25 +201,24 @@ def get_table_knowledge_graph(self) -> KnowledgeGraph:
             # now, and later add number and string entities as needed.
             number_columns = []
             date_columns = []
-            for column, column_type in self.column_types.items():
-                if column_type == "number":
-                    column_name = f"number_column:{column}"
-                    number_columns.append(column_name)
-                elif column_type == "date":
-                    column_name = f"date_column:{column}"
-                    date_columns.append(column_name)
-                else:
-                    column_name = f"string_column:{column}"
+            for typed_column_name in self.table_data[0].keys():
```

Comment:
Don't you have a `column_names` field that you can use instead of `self.table_data[0]`?

Reply:
Yes, thanks!